api/envoy/extensions/upstreams/http/v3/http_protocol_options.proto

```diff
@@ -57,10 +58,10 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 //        .... [further cluster config]
 message HttpProtocolOptions {
   // If this is used, the cluster will only operate on one of the possible upstream protocols (HTTP/1.1, HTTP/2).
-  // If :ref:`http2_protocol_options <envoy_api_field_config.cluster.v3.Cluster.http2_protocol_options>` are
-  // present, HTTP2 will be used, otherwise HTTP1.1 will be used.
   message ExplicitHttpConfig {
     oneof protocol_config {
+      option (validate.required) = true;
```

Comment:
While you are in here can you potentially make it more clear the http2 needs to be used for gRPC? I think that would help guide people in the right way even if we don't want to do something extreme like `http2_and_grpc_protocol_options` or something like that, which we probably don't want to do because h3 can be used with gRPC. So maybe just some comments/docs?